package-lock.json

```diff
@@ -41797,7 +41797,7 @@
     },
     "packages/remark-plugins": {
       "name": "@hashicorp/platform-remark-plugins",
-      "version": "0.2.0",
+      "version": "0.2.1",
```

Comment:
how did this get bumped as part of this PR?

Reply:
whoops, this is muscle memory of `git pull && npm i`.